### ISSUE TITLE
Name or label for node

### ISSUE THREAD

**Issue by mrunks** [Blazor-Diagrams/Blazor.Diagrams#123]
I am unable to find a way to add a name or label over the default label for node without creating a custom model and widget.  Is there a standard property that exists or could be added to simply set the label when the node is rendered?

**Comment by zHaytam** [comment]
Hi, unfortunately, there isn't. May I know why you need it? The default node is only there as an example, I didn't think anyone would use them as is.

**Comment by mrunks** [comment]
I am trying upgrade a project at my company that is using an older version of JS Plumb where we used the standard icon and added a name and added anchors just like your code does.  The one simple missing item just slapping a name on the standard node. 

I looked at your custom node and custom node link example  I recreated the custom node pieces but now It does not render the anchors any longer.  I did not use the custom node link.  For our project the default out of the box node shape is perfectly fine for what we do and there would be no need to render a custom one.

Thanks,

**Comment by zHaytam** [comment]
That's weird, may I see your component's code?

Anyway, I will try to add this option as soon as I can.

**Comment by mrunks** [comment]
Sorry I made a mistake when I said anchors, I meant to say it no longer renders the "Ports" such as Portalignment.bottom, PortAlignment.Top etc.    

Is there a way to have the default ports still show up even if I was to use the custom DiagramNodeWidget as in your example?  

Regardless, having the option to simply add a name to the default node would be great.  Thank you.

**Comment by zHaytam** [comment]
I will add it in the next version, don't worry. In the meantime, use:

```
<div class="default-node @(Node.Group != null ? "grouped" : "") @(Node.Selected ? "selected" : "")">
    @Node.Title
    @foreach (var port in Node.Ports)
    {
        <PortRenderer Port="port" Class="default"></PortRenderer>
    }
</div>

@code {
    [Parameter]
    public MyNodeModel Node { get; set; }
}
```

With a model like:
```
public class MyNodeModel  : NodeModel
{
        public MyNodeModel (string title, Point position = null) : base(position)
        {
                Title = title;
        }

        public string Title { get; set; }
}
```

**Comment by mrunks** [comment]
Thanks.  I will use your example for now to render custom ports.

**Comment by zHaytam** [comment]
I have added a `Title` property in the nodes, it will be available in the next version.
Since the default node uses CSS Flex, you can align the title however you want using for example:
````css
justify-content: end;
align-items: end;
padding: 10px; /* to avoid ports */
```

Hope this helps!

**Comment by zHaytam** [comment]
This is now available in 2.1.4!
